utils/build-script

```diff
@@ -26,30 +31,47 @@ from build_swift.build_swift import presets
 
 import six
 
-from swift_build_support.swift_build_support import (
-    diagnostics,
-    products,
-    shell,
-    targets,
-    workspace
-)
+from swift_build_support.swift_build_support import diagnostics
+from swift_build_support.swift_build_support import products
+from swift_build_support.swift_build_support import shell
+from swift_build_support.swift_build_support import targets
+from swift_build_support.swift_build_support import workspace
 from swift_build_support.swift_build_support.SwiftBuildSupport import (
     HOME,
     SWIFT_BUILD_ROOT,
     SWIFT_REPO_NAME,
-    SWIFT_SOURCE_ROOT,
-)
+    SWIFT_SOURCE_ROOT)
```

Comment:
I think that this hunk should be dropped, the trailing comma is useful

Reply:
I intend to move all these variables to `build_swift` in the _very_ near future, trailing comma or not.

Reply:
Sure, but, I think that preserving the comma as it moves is still valuable.